apps/web/components/security/EnableTwoFactorModal.tsx

```diff
@@ -159,24 +160,20 @@ const EnableTwoFactorModal = ({ onEnable, onCancel }: EnableTwoFactorModalProps)
           </>
         </WithStep>
         <WithStep step={SetupStep.EnterTotpCode} current={step}>
-          <form onSubmit={handleEnable}>
+          <form className=" flex justify-center" onSubmit={handleEnable}>
             <div className="mb-4">
               <label htmlFor="code" className="mt-4 block text-sm font-medium text-gray-700">
                 {t("code")}
               </label>
               <div className="mt-1">
-                <input
-                  type="text"
-                  name="code"
-                  id="code"
-                  required
+                <OtpInput
```

Comment:
Why not use the [one we already have](https://github.com/calcom/cal.com/blob/main/apps/web/components/auth/TwoFactor.tsx#L9)?

Reply:
I tried to use it at first, but had some hard deps on using rhf with it, and I was not using a form already?

Should I move it to the one we already have and add the necessary forms if needed?

Reply:
Yes, let's keep it consistent regardless of what we're using

Reply:
Done, now we reuse TwoFactor and added react hook forms on were we use it